src/lib.rs

```diff
@@ -704,6 +704,52 @@ enum InputElementDiv {
     DecimalDigit(DecimalDigit),
 }
 
+#[derive(Debug, FromPest)]
+#[pest_ast(rule(lexical::Rule::InputElementRegExp))]
+enum InputElementRegExp {
+    WhiteSpace(WhiteSpace),
+    LineTerminator(LineTerminator),
+    Comment(Comment),
+    CommonToken(CommonToken),
+    RightBracePunctuator(RightBracePunctuator),
+}
+
+#[derive(Debug, FromPest)]
+#[pest_ast(rule(lexical::Rule::InputElementRegExpOrTemplateTail))]
+enum InputElementRegExpOrTemplateTail {
+    WhiteSpace(WhiteSpace),
+    LineTerminator(LineTerminator),
+    Comment(Comment),
+    CommonToken(CommonToken),
+    DivPunctuator(DivPunctuator),
+}
+
+#[derive(Debug, FromPest)]
+#[pest_ast(rule(lexical::Rule::InputElementHashbangOrRegExp))]
+enum InputElementHashbangOrRegExp {
+    WhiteSpace(WhiteSpace),
+    LineTerminator(LineTerminator),
+    Comment(Comment),
+}
+
+enum PackedToken {
```

Comment:
```suggestion
#[allow(dead_code)]
enum PackedToken {
```

Ignore linter warnings about currently unused enum members:

```plain
error: variants `InputElementHashbangOrRegExp`, `InputElementRegExp`, and `InputElementRegExpOrTemplateTail` are never constructed
   --> src/lib.rs:737:5
    |
735 | enum PackedToken {
    |      ----------- variants in this enum
736 |     InputElementDiv(InputElementDiv),
737 |     InputElementHashbangOrRegExp(InputElementHashbangOrRegExp),
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
738 |     InputElementRegExp(InputElementRegExp),
    |     ^^^^^^^^^^^^^^^^^^
739 |     InputElementRegExpOrTemplateTail(InputElementRegExpOrTemplateTail),
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `-D dead-code` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(dead_code)]`
```